backend/src/middleware/notifications/extractMentions.spec.js

```diff
@@ -0,0 +1,46 @@
+import extractIds from './extractMentions'
+
+describe('extract', () => {
+  describe('searches through links', () => {
+    it('ignores links without .mention class', () => {
+      const content = '<p>Something inspirational about <a href="/profile/u2" target="_blank">@bob-der-baumeister</a> and <a href="/profile/u3" target="_blank">@jenny-rostock</a>.</p>'
+      expect(extractIds(content)).toEqual([])
+    })
+
+    describe('given a link with .mention class', () => {
+      it('extracts ids', () => {
```

Comment:
I pulled down the code and added a went to `/post/create`, and was playing around with the updated `Editor`, it seems really nice to me.

One thing I did was to add a mention, and then inspected and looked at the link and it doesn't seem to be added a `.mention` class as I was suspecting based on the tests, but maybe this is removed in `xssMiddleware` as your comment :point_down: suggests?
```
<a href="/profile/u2" target="_blank">@bob-der-baumeister</a>
```
![Screenshot at 2019-04-17 14:15:33](https://user-images.githubusercontent.com/26943915/56308180-03f75f00-611d-11e9-9345-06f00b7d8c82.png)
![Screenshot at 2019-04-17 14:28:50](https://user-images.githubusercontent.com/26943915/56308242-2e491c80-611d-11e9-9caf-fa21e47ede5d.png)